### DISCUSSION TITLE
Remove duplicate result from the result set

### DISCUSSION THREAD

**Discussion by ishanuda** [krisk/Fuse#700] [Q&A] [answered]
Hi,
In a search content, if a word exist with multiple occurrences, the search result is cumming with duplicate result items in the result array pointing to the same content.

I wold like to remove the additional result from the

**Comment by krisk** [comment, marked as answer]
Late reply — sorry about that.

Fuse shouldn't return duplicate result items — each document appears at most once in the results, with all its matches aggregated. If you're seeing duplicates, it might be that your data contains duplicate entries. Could you share a minimal example that reproduces this?
